backend/requirements.txt

```diff
@@ -17,27 +22,34 @@ gunicorn==20.1.0
 idna==2.10
 itsdangerous==2.0.1
 Jinja2==3.0.1
+jsonschema==4.1.0
 Mako==1.1.4
 MarkupSafe==2.0.1
+marshmallow==3.14.0
```

Comment:
Do you use marshmallow anywhere? If not please uninstall it and remove it from the requirements file.